clang-tools-extra/clang-include-fixer/find-all-symbols/STLPostfixHeaderMap.cpp

```diff
@@ -15,9 +15,11 @@ const HeaderMapCollector::RegexHeaderMap *getSTLPostfixHeaderMap() {
   static const HeaderMapCollector::RegexHeaderMap STLPostfixHeaderMap = {
       {"include/__stdarg___gnuc_va_list.h$", "<cstdarg>"},
       {"include/__stdarg___va_copy.h$", "<cstdarg>"},
+      {"include/__stdarg_macro.h$", "<cstdarg"},
       {"include/__stdarg_va_arg.h$", "<cstdarg>"},
       {"include/__stdarg_va_copy.h$", "<cstdarg>"},
       {"include/__stdarg_va_list.h$", "<cstdarg>"},
+      {"include/__stddef_macro.h$", "<cstddef"},
```

Comment:
Looks like the closing `>` is missing for these two.

Reply:
Oh, dangit, thanks.